auto-value-gson/build.gradle

```diff
@@ -5,16 +5,19 @@ sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
 dependencies {
-    compile 'com.google.code.gson:gson:2.8.0'
+    compile project(':auto-value-gson-annotations')
+    compile deps.gson
     compile 'com.squareup:javapoet:1.8.0'
-    compile 'com.google.auto.value:auto-value:1.3'
+    compile 'com.jakewharton.auto.value:auto-value-annotations:1.4'
+    compileOnly 'com.google.auto.value:auto-value:1.4'
```

Comment:
Sure, this artifact is then used as `annotationProcessor` dependency later... 
I think `compileOnly` for `auto-service` made me wrong... Thanks for the comment.